.github/workflows/pr-labeling.yml

```
@@ -16,34 +16,33 @@ jobs:
       contents: read
       pull-requests: write
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-
       - name: Label PR based on files changed
         uses: actions/labeler@v4
```

actionlint (1.7.11)

[error] the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)